src/osemgrep/reporting/Cli_json_output.ml

```diff
@@ -227,38 +227,15 @@ let cli_error_of_core_error (x : OutJ.core_error) : OutJ.cli_error =
 (*****************************************************************************)
 (* LATER: we should get rid of those intermediate Out.core_xxx *)
 
-let make_fixed_lines ?applied_fixes lines fix path (start : OutJ.position)
+let make_fixed_lines fixes_env fix path (start : OutJ.position)
```

Comment:
We could also probably just get rid of this function now

Reply:
I thought about it, but I slightly prefer encapsulating the logic to create the textedit. Since it sounds like you don't feel strongly I'll leave it in, but let me know if I've misunderstood.